resources/src/commands/onboard_user.rs

```diff
@@ -0,0 +1,32 @@
+use crate::commands::{chat_post_message, conversations_open};
+use serde_json::{json, Value};
+
+// TODO: write docs
+pub async fn run(body: &Value) {
+    let username: &str = body["event"]["user"]["id"]
+        .as_str()
+        .unwrap_or("invalid_user_name");
+    let first_name: &str = body["event"]["user"]["profile"]["first_name"]
+        .as_str()
+        .unwrap_or("");
+    let text: String = format!(":codedevils_flash: Hey welcome to CodeDevils {} :codedevils_flash:. I am DevilBot and I don't know much yet, but here's what I do know: \
+        This Slack workspace serves as the main communication platform for all things CodeDevils :partywizard:. All our announcements can be found in the <#C30L07P18> channel. \
+        This includes all meetings and meeting recordings! I'd like you to go to the <#CMGU8033K> channel and introduce yourself. After that, come on over to\
+        <#C2N5P84BD>. Most of my creators are there all day.", &first_name);
```

Comment:
This function should generalize the these parameters better. You should definitely be passing in the onboarding message. The body should be parsed somewhere else and you just pass in the required params (e.g. `username`, `first_name`, and `channel_id`) since those params are going to need to be parsed for multiple different functions.

Reply:
I do something similar in [dynamo.rs](https://github.com/ASU-CodeDevils/devil_bot_rust/blob/main/resources/src/aws/dynamo.rs) for reference.

Reply:
Will do

Reply:
Before I resolve this, you won't see `channel_id` as a param because `channel_id` comes from opening a conversation between DB and the user that has joined the workspace